Celeste.Mod.mm/Patches/DeathEffect.cs

```diff
@@ -1,24 +1,50 @@
-using System;
+using System;
+using System.Xml;
 using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
 using Mono.Cecil;
 using Mono.Cecil.Cil;
+using Monocle;
 using MonoMod;
-using MonoMod.Cil;
+using MonoMod.Cil;
+using MonoMod.InlineRT;
+using MonoMod.Utils;
 
 namespace Celeste {
     class patch_DeathEffect : DeathEffect {
 
+        // no-op. MonoMod ignores this - we only need this to make the compiler shut up.
         public patch_DeathEffect(Color color, Vector2 offset)
             : base(color, offset) { }
 
+
+        public MTexture Texture;
+
+        /// <summary>
+        /// a temporary texture used to the next DeathEffect.Draw get call
+        /// </summary>
+        public static MTexture _texture;
```

Comment:
This shouldn't be public

Reply:
I had public it for do some special testing by other mods, I must have forgotten to change it back...